System.IO.Abstractions.TestingHelpers/MockDirectory.cs

```diff
@@ -378,6 +378,14 @@ public override void Move(string sourceDirName, string destDirName)
                 throw new IOException("Source and destination path must be different.");
             }
 
+            //if we're moving a file, not a directory, call the appropriate file moving function.
+            var fileData = mockFileDataAccessor.GetFile(fullSourcePath);
+            if (fileData != null && (fileData.Attributes & FileAttributes.Directory) == 0)
```

Comment:
I think a safe navigation operator (?.) might simplify this conditional a little?

Reply:
Sure, changed this line to use safe navigation operator and `Enum.HasFlag`. Reads a lot easier now.